include/m_common_data.h

```diff
@@ -29,6 +29,15 @@ typedef UNK_PTR (*CommonData_unk_1004C_unk_0C)(size_t, const struct struct_80116
 typedef UNK_RET (*CommonData_unk_1004C_unk_10)(struct Actor*);
 typedef UNK_RET (*CommonData_unk_1004C_unk_14)(struct CommonData_unk_1004C_unk_14_arg0*, u16);
 
+typedef UNK_RET (*CommonData_unk_1004C_unk_BC)(struct Actor*, struct Game_Play*);
+typedef void (*CommonData_unk_1004C_unk_C0)(struct Actor*, struct Game_Play*, void*);
```

Comment:
```suggestion
typedef void (*CommonData_unk_1004C_unk_C0)(struct Actor*, struct Game_Play*, struct struct_809AEFA4*);
```

You'll also need to put a `struct struct_809AEFA4;` at the top of this file

Reply:
It may not be shared?